src/testUtil/serialize.ts

```diff
@@ -1,41 +1,39 @@
-// @ts-nocheck
 // From https://github.com/nodeca/js-yaml/issues/586#issuecomment-814310104
```

Comment:
Epic!  Presumably you tried running this?  I don't believe it is actually tested.  Best test is prob to run the "Update snapshots" debug launch config in VSCode

Reply:
Huh, it seems like I may have accidentally made `documentContents` serialise as an inline string. I'll look into that.

Reply:
Oh wait, that might just be because I have hard tabs by default, so the YAML serialiser wants to serialise a tab character as `"\t"`:
```yaml
documentContents: "try {\n\tif (true) {\n\t    const foo = \"hello\";\n\t}\n} catch (err) {\n\t\n}\n\ntry {\n\tconst bar = \"hello\";\n} catch (err) {\n\t\n}"
```

Reply:
Yeah probably.  Might be worth just changing your local setting temporarily so we don't have to wait on #731?

Reply:
Ok well it has nothing to do with what you've done in this PR, so I'd consider this a passing test from that perspective.

As for why it's doing that, idk that's weird.  Are tests passing for you locally?

Reply:
Oddly enough they are passing locally after blowing away those changes. 🤔

Reply:
Strange.  Possible that test fails nondeterministically.  We retry all tests 5 times, but when you do a snapshot update it will just use the first result so any failure will just become a snapshot update and not re-run